CONTRIBUTING.md

```diff
@@ -10,33 +10,46 @@ Welcome to the contributing guide for Virtual Coffee's Preptember repo! Thank yo
 
 We take our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/) very seriously, and all contributors must abide by it. Let’s treat each other with respect, and remember, we are all people who care and are trying. If you believe someone has violated the Code of Conduct, please fill out our [CoC Violation Form](https://virtualcoffee.io/report-coc-violation/), which you can do anonymously.
 
-## New contributors
+## New Contributors
 
 If you're new to open source, check out the resources below to help you familiarize yourself with open source:
 
 - [Virtual Coffee's Guide to Open Source](https://virtualcoffee.io/resources/developer-resources/open-source)
-- [Intro to Open Source with OpenSauced](https://intro.opensauced.pizza/#/)
+- [Intro to Open Source with OpenSauced](https://opensauced.pizza/learn/intro-to-oss)
 - [Git and GitHub learning resources](https://docs.github.com/en/get-started/quickstart/git-and-github-learning-resources)
 - [Git Guides](https://github.com/git-guides)
 
-## Contribution type
+## Contribution Type
 
 There are two types of contributions to this repo:
 
-1. 🌱 **Practicing Open Source** — Add your name, link to your GitHub profile, and your favorite coffee (or any other drink you like!) to the `preptember-participants.md` file.
+1. 🌱 **Practicing Open Source** — Add your name, link to your GitHub profile, and your favorite cake or snack to the `preptember-participants.md` file.
 
-2. 📃 **Repositories List** — Add the link to a repository(ies) you recommend for Hackoberfest to the `repositories-list.md` file _only_ after the repository meets our requirements — check our [repository checklist](https://virtualcoffee.io/resources/developer-resources/open-source/maintainer-guide#repository-checklist). You can always return and add more repositories here.
+2. 📃 **Repositories List** — Add the link to a repository(ies) you recommend for Hacktoberfest to the `repositories-list.md` file _only_ after the repository meets our requirements — check our [repository checklist](https://virtualcoffee.io/resources/developer-resources/open-source/maintainer-guide#repository-checklist). You can always return and add more repositories here.
```

Comment:
**suggestion (wording, non-blocking):** Reword slightly for clarity

```suggestion
2. 📃 **Repositories List** — Add the name and link to a repository you recommend for Hacktoberfest to the `repositories-list.md` file. Do this **_only after_** the repository meets our requirements — see our [repository checklist](https://virtualcoffee.io/resources/developer-resources/open-source/maintainer-guide#repository-checklist). You can add multiple repositories here!
```

Reply:
I really like this!

However, I want to also mention that after their first PR, folks may come back and create another PRs when they find more repos along the way.

Any idea on how to reword that?

Reply:
@adiati98 @meg-gutshall How about this for the last sentence:

> You can add multiple repositories here and you can always make another PR if you have more for us later in the month!